src/test/java/bio/terra/common/logging/LoggingTest.java

```diff
@@ -81,8 +77,7 @@
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
         throws IOException, ServletException {
-      Scope scope =
-          Tracing.getTracer().spanBuilderWithExplicitParent("test-span", null).startScopedSpan();
+      Tracing.getTracer().spanBuilderWithExplicitParent("test-span", null).startScopedSpan();
       requestSpanContext = Tracing.getTracer().getCurrentSpan().getContext();
       chain.doFilter(request, response);
       // Note: we purposefully don't close the scope here. For some reason, it doesn't seem possible
```

Comment:
This sounds odd. Possibly out of scope.

Reply:
:) it's a condensation of a much more frustrating debugging session than one might imagine...